<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewRemindDialog</class>
 <widget class="QDialog" name="NewRemindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Duration</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Expires</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Repeatable</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="syncPushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/update.ico</normaloff>:/icons/update.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="durationDays">
       <property name="suffix">
        <string>d</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="durationHours">
       <property name="suffix">
        <string>h</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="durationMinutes">
       <property name="suffix">
        <string>m</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="durationSeconds">
       <property name="suffix">
        <string>s</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QTextEdit" name="descriptionLineEdit"/>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="5" column="1" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewRemindDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewRemindDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
